docs/source/using/addons.adoc

```diff
@@ -57,26 +64,35 @@ To do so you can specify the optional _OpenShift-Version_ metadata field. The se
 | >=3.5.0, <=3.8.0 | OpenShift version should higher or equal to 3.5.0 but lower or equal to 3.8.0.
 |===
 
-NOTE: If the metadata field _OpenShift-Version_ is not specified in the add-on header, the add-on is applied against any version of OpenShift.
+[NOTE]
+====
+If the metadata field *OpenShift-Version* is not specified in the add-on header, the add-on is applied against any version of OpenShift.
```

Comment:
Shouldn't it be s/the add-on is applied against any/the add-on can be applied against any/

Reply:
Agreed, that makes more sense to me.